test/libsolidity/semanticTests/abiEncoderV2/abi_decode_fixed_arrays.sol

```diff
@@ -0,0 +1,12 @@
+pragma experimental ABIEncoderV2;
```

Comment:
Didn't we have a test option `abiencoderv2: also` or something like that? If yes, it could be used in the other file and this and the other ABIEncoderV2 test files wouldn't be necessary

Reply:
I'll check now

Reply:
Actually, there is `ABIEncoderV1Only` whiche defaults to false. So, I can remove the tests from V2 since there is CI job that forces encoder v2.

Reply:
Removed tests that were basically duplicated

Reply:
Ah ok!